### DISCUSSION TITLE
ERROR

### DISCUSSION THREAD

**Discussion by hanabarak** [bio-miga/miga#179] [General]
In the last few days, when trying to upload a new genome to "NCBI Prok" the following message was accepted:

We're sorry, but something went wrong.
If you are the application owner check the logs for more information.

![t](https://github.com/bio-miga/miga/assets/28673144/7846e8a7-aa41-4de1-9382-eb99f785e28e)

**Comment by lmrodriguezr** [comment]
Dear @hanabarak ,

Thank you for your report. The servers were down for maintenance and a connection failed to reload. Service has now been reestablished, please try again.

Also, please note that a new MiGA server with up-to-date databases is also available at:
https://uibk.microbial-genomes.org/

Best wishes,
Miguel.
